cmd/docker-app/render.go

```diff
@@ -69,6 +73,15 @@ func renderCmd(dockerCli command.Cli) *cobra.Command {
 	cmd.Flags().StringArrayVarP(&renderSettingsFile, "settings-files", "f", []string{}, "Override settings files")
 	cmd.Flags().StringArrayVarP(&renderEnv, "set", "s", []string{}, "Override settings values")
 	cmd.Flags().StringVarP(&renderOutput, "output", "o", "-", "Output file")
+	cmd.Flags().StringVarP(&renderDisable, "disable", "d", "", "Disable services listed as comma separated values")
```

Comment:
I think it'd make more sense to use `StringArrayVarP` here.

Reply:
Sure! Good catch!